src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMessage.cs

```diff
@@ -492,8 +550,21 @@ public int Encode(Span<byte> destination)
             return bytesWritten;
         }
 
+        /// <summary>
+        /// When overriden in a derived class, encodes this <see cref="CoseMessage"/> as CBOR.
+        /// </summary>
+        /// <param name="destination">The buffer in which to write the encoded value.</param>
+        /// <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination" />.</param>
```

Comment:
```suggestion
        /// <param name="bytesWritten">On success, receives the number of bytes written to <paramref name="destination" />.  This parameter is treated as uninitialized.</param>
```

Per the `out` row at https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors